app/ide-desktop/lib/dashboard/src/layouts/SettingsSidebar.tsx

```diff
@@ -133,19 +134,22 @@ export default function SettingsSidebar(props: SettingsSidebarProps) {
               >
                 {section.name}
               </aria.Header>
-              {section.tabs.map(tab => (
-                <SidebarTabButton
-                  key={tab.settingsTab}
-                  isDisabled={(tab.organizationOnly ?? false) && !isUserInOrganization}
-                  id={tab.settingsTab}
-                  icon={tab.icon}
-                  label={tab.name}
-                  active={tab.settingsTab === settingsTab}
-                  onPress={() => {
-                    setSettingsTab(tab.settingsTab)
-                  }}
-                />
-              ))}
+
+              <ariaComponents.ButtonGroup gap="xxsmall" direction="column" align="start">
```

Comment:
please try to avoid changing layout from the figma design. (the figma design does not have a gap)

Reply:
Gaps are intended because, even though it's not described in the design if we remove the gaps, menu items will look stuck together(see screenshot). Alternatively, we can use a different hover for elements.

![CleanShot 2024-05-23 at 13 58 45](https://github.com/enso-org/enso/assets/61194245/58d5037f-9c1c-465c-8451-05f01dac5542)

Reply:
alternatively reduce by just 1px or 0.5px to make the gaps consistent with those ofnthe assets table rows